convene-web/app/helpers/rooms_helper.rb

```diff
@@ -14,4 +14,8 @@ def video_room_path(workspace, room)
       workspace_room_path(room.workspace, room)
     end
   end
+
+  def video_room_name(workspace, room)
+    "#{workspace.slug}--#{room.slug}"
+  end
```

Comment:
Could we put this on the `Room` class, perhaps with a method name like `cross_system_unique_identifer` or something to indicate that it's used to figure out how to reference the particular room across Jitsi/DOM/etc? It's almost like a URI but not quite...

Also, are the two dashes semantically meaningful? My understanding of the direction that semantic HTML/CSS are trending towards is that: ` ` or `_` seperates ideas, `-` seperates words, `--` gives you a state.

Reply:
Apparently this is called "block element modifier" http://getbem.com/

Reply:
Yes and great name suggestion.